test/results_test.exs

```diff
@@ -58,13 +68,118 @@ defmodule Wanda.ResultsTest do
                %ExecutionResult{
                  execution_id: ^execution_id,
                  group_id: ^group_id,
-                 payload: %{
-                   "result" => "passing",
-                   "check_results" => [_ | _]
-                 }
+                 status: :running,
+                 payload: %{},
+                 targets: [
+                   %{
+                     agent_id: ^agent_id_1,
+                     checks: ^checks_1
+                   },
+                   %{
+                     agent_id: ^agent_id_2,
+                     checks: ^checks_2
+                   }
+                 ]
                }
              ] = Repo.all(ExecutionResult)
     end
+
+    test "should raise an error when trying to create an already existing execution" do
+      [
+        %ExecutionResult{execution_id: execution_id, group_id: group_id},
+        %ExecutionResult{}
```

Comment:
What is this empty execution result for?

Reply:
It is not really empty, it is the second element of what we get from `insert_list(2, :execution_result)`, I am interested that there is a second element, yet I don't mind what it carries with it.

I can completely ignore it this way
```elixir
[
  %ExecutionResult{execution_id: execution_id, group_id: group_id},
  _
] = insert_list(2, :execution_result)
```

Reply:
The question was more about why we need it hehe
As far as I see it is only used to check later on that we have 2 rows stored, which is not really needed. You could simply check that you have one, and forget about this one, as it doesn't really add any value.
Anyway, it is a trivial thing, so don't bother that much